src/bp/core/module-loader.ts

```diff
@@ -330,7 +330,7 @@ export class ModuleLoader {
       .filter(module => module.botTemplates)
       .map(module => {
         return module.botTemplates!.map(template => {
-          return { ...template, moduleId: module.definition.name, moduleName: module.definition.fullName }
+          return { ...template, moduleName: module.definition.name }
```

Comment:
Can you add the fullName property so it's what is displayed in the bot template list? 

![image](https://user-images.githubusercontent.com/42552874/98700407-24fa5300-2346-11eb-9c12-518eca5781da.png)

Reply:
Good catch, I've just added the module's full name.

<img width="694" alt="Screen Shot 2020-11-12 at 10 25 22 AM" src="https://user-images.githubusercontent.com/1047845/98959834-afbb8900-24d1-11eb-82f3-50d7ba31a49f.png">